C++/Indentation Rules.tmPreferences

```diff
@@ -23,7 +23,7 @@
 		<key>bracketIndentNextLinePattern</key>
 		<string>(?x)
 		^ \s* \b(if|while|else)\b [^;]* $
-		| ^ \s* \b(for)\b .* $
+		|   ^ \s* \bfor\b (?:[^;]++|;[ ]+(?!$))+ \s* $
```

Comment:
This rule breaks intention, even if the `;` is part of the comment, but this seems the case for most such rules.

```c
for(i = 0; dd < 5; i++) // comment ;
// no indention applied here!!
```